src/transformers/models/superglue/modeling_superglue.py

```diff
@@ -0,0 +1,845 @@
+# Copyright 2024 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""PyTorch SuperGlue model."""
+from copy import deepcopy
+from dataclasses import dataclass
+from typing import List, Optional, Tuple, Union
+
+import torch
+from torch import nn
+
+from transformers import PreTrainedModel, add_start_docstrings
+from transformers.models.superglue.configuration_superglue import SuperGlueConfig
+
+from ...utils import (
+    ModelOutput,
+    add_start_docstrings_to_model_forward,
+    logging,
+)
```

Comment:
```suggestion
from ...utils import ModelOutput, add_start_docstrings_to_model_forward, logging
```

Reply:
Fixed in [this commit](https://github.com/huggingface/transformers/pull/29886/commits/601da1cf0756cef7339d4dd33faeefbfcd073ac9)